README.md

```diff
@@ -23,3 +34,13 @@ cd gnu-koyravov
 
 ```
 
+### Конфигурация
+
+Текущий набор скриптов позволяет открыть Корявник лишь с помощью [Okular](https://github.com/KDE/okular), однако интегрировать свой вариант очень легко. Нужно в соответсвующем месте `open.sh` заменить вызов `okular.sh` на требуемый скрипт.
```

Comment:
It is better to put all configurations made by user in either .json or .yml config file and then parse it with the script.

Reply:
Agree. I'll use config file then